doc/nrf/releases_and_maturity/migration/migration_guide_2.8.rst

```diff
@@ -1,3 +1,5 @@
+:orphan:
+
```

Comment:
For Wi-Fi cred:

```
+Wi-Fi®
+------
+
+.. toggle::
+
+   * For :ref:`lib_wifi_credentials` library:
+
+     * Syntax for ``add`` command has been modified to support ``getopt`` model.
+         For example, the following command with old syntax:
+         ``wifi_cred add SSID WPA2-PSK password`` should be replaced with the following command with new syntax:
+         ``wifi_cred add -s SSID -k 1 -p password``.
+         ``wifi_cred add --help`` command will provide more information on the new syntax.
+
```

Reply:
Was already added to the Migration guide.